draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1429,6 +1290,150 @@ RoQ, but are not required by RoQ.
   certain frame that should be cancelled before resuming transmission of further
   frames on new QUIC streams.
 
+# Considered RTCP Packet Types and RTP Header Extensions
+
+This section lists all the RTCP packet types and RTP header extensions that were
+considered in the analysis described in {{rtcp-mapping}}.
+
+## RTCP Control Packet Types {#control-packets}
+
+Several but not all of these control packets and their attributes can be mapped
+from QUIC, as described in {{transport-layer-feedback}}. *Mappable from QUIC*
+has one of three values: *yes*, *QUIC extension required*, and *no*.
```

Comment:
Two things for this text: 

> Several but not all of these control packets and their attributes can be mapped
> from QUIC, as described in {{transport-layer-feedback}}. *Mappable from QUIC*
> has one of three values: *yes*, *QUIC extension required*, and *no*.

I THINK this paragraph applies to more subsections than **## RTCP Control Packet Types {#control-packets}**.

Also, I'm also seeing **partly** and **possibly** in this section, but I'm not seeing **QUIC extension required**.

Reply:
Thanks! I moved the paragraph to the section introduction so it applies to all the following subsections. I also replaced *QUIC extension required* with *possibly* and *partly*, including short explanations and updated the table values to match their respective descriptions.